README.md

```diff
@@ -1,41 +1,67 @@
-# fastly-tools
+# fastly-tools [![CircleCI](https://circleci.com/gh/Financial-Times/fastly-tools.svg?style=svg&circle-token=33bcf2eb98fe2e875cc66de93d7e4a50369c952d)](https://circleci.com/gh/Financial-Times/fastly-tools)
+
+This library is a command line tool for interacting with the FT.com CDN, [Fastly](https://www.fastly.com/).
+
+
+## Requirements
+
+* Node 5.x
 
-Command Line Utility for interacting with fastly
 
 ## Installation
 
-```
-npm install --save-dev @financial-times/fastly-tools
+```sh
+git clone git@github.com:Financial-Times/fastly-tools.git
+cd fastly-tools
+make install
 ```
 
-## Usage
 
-	fastly [options] [command]
+## Development
 
-      Commands:
+### Testing
 
-        deploy [options] [folder]
-           Deploys VCL in [folder] to the specified fastly service.  Requires FASTLY_APIKEY env var
+In order to run the tests locally you'll need to run:
 
-      Options:
+```sh
+make test
+```
 
-        -h, --help  output usage information
+### Install from NPM
 
+```sh
+npm install --save-dev @financial-times/fastly-tools
+```
+
+### Usage
 
-## Commands
+```sh
+Usage: fastly [options] [command]
 
-### deploy
-Deploys VCL in [folder] to the specified fastly service.  Requires FASTLY_APIKEY env var
+Options:
+  -h, --help                 output usage information
+
+Commands:
+  deploy [options] [folder]  Deploys VCL in [folder] to the specified fastly service.  Requires FASTLY_KEY env var
+```
 
-  Usage: deploy [options] [folder]
+### Deploy
 
-  Options:
+Deploys VCL in [folder] to the specified fastly service. Requires FASTLY_APIKEY environment variable.
```

Comment:
This is a repeat of the sentence given in the code block so we can probably do without it here.

There is also a difference of names of the key value requires: `FASTLY_APIKEY` here but `FASTLY_KEY` below - which one is the correct name?

Also: where can I acquire the `FASTLY_APIKEY/FASTLY_KEY` env var value? Presumably in the Vault directory for this repo, but might be useful to state this explicitly.

Reply:
ive just checked the code and `FASTLY_APIKEY` is the correct one, good catch!

I'm not too sure where the key is found, i'll have a dig around.

Reply:
@andygout I have found that each project that uses this tool to deploy will hold their own API key for Fastly so its not 1 ring, i mean key, to rule them all 🙂

Reply:
…and in the darkness bind them.

So as an example, if `next-foobar` were to use this tool to deploy, it would have its own API key for Fastly - where would that API key be stored?

Reply:
It would be in vault, for example https://vault.in.ft.com:8080/ui/vault/secrets/secret/show/teams/next/next-api/development

Reply:
Ah, nice - I see it. Personally, having this pointer would be really instructive - would you be able to add a description and/or example of the key's location?